### PR TITLE
added displayCallbackCounter function for testing purpose

### PR DESCRIPTION
**Pull request by msakowicz**

### CODE REVIEW

File: example/index.html [hunk 1]
```diff
@@ -130,6 +130,37 @@ <h1 class="example-form__title">
         });
     }
 
+    var callbackCounter = 0;
```
Comment (pgs-gmartowski) [inline example/index.html:133]: you can easily move this script to separate .ts file; there are too many js lines in index.html

Comment (pgs-gmartowski) [reply example/index.html:133]: @msakowicz continuation in https://github.com/SecureTrading/js-payments/pull/1249

**Comment by wirus15** [comment]
@msakowicz isn't this alread available in this branch https://github.com/SecureTrading/js-payments/pull/1249 ?

**Comment by msakowicz** [comment]
> @msakowicz isn't this alread available in this branch #1249 ?

@wirus15  yes, it is :) I will delete this branch

**Pull request closed without merging**